### PR TITLE
[SEMI-MODULAR][READY] Recreates the admin verb 'Create Antagonist(s)'

### PR DESCRIPTION
**Pull request by ZephyrTFA**
THIS MODIFIES ADMIN TOPIC.DM

This, surprisingly, only took like ~~six~~ ten hours of coding.

## About The Pull Request

After much pain, much anger, much frustration, and about two close-calls with my temper; I present to you a recreation of the old Create Antagonist verb!

## Why It's Good For The Game

I don't know; I was asked by @hugmeorelse to add this back.

## Changelog
:cl: ZephyrTFA
admin: Create Antagonist is back! Admemes rejoice!
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
Not a prio, but creating wizards would be good to have as an option

**Comment by ZephyrTFA** [comment]
> 
> 
> Not a prio, but creating wizards would be good to have as an option

It exists. It iterates over all glob.special_roles; and then ignores PAI, SENTIENCE, MALF

**Comment by tf-4** [comment]
hhh teaches me to comment before I finish reading

**Comment by hugmeorelse** [comment]
I love you, @GoldenKeyboard

**Comment by ZephyrTFA** [comment]
Alright; I'm fairly certain this is ready to merge now; but I'm still open to requests for it. Except Obsessed, I cannot for the life of me get it to work so I'm just called it intended that you can't make them.

**Comment by ZephyrTFA** [comment]
When the FUCK did I delete the branch

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -47,6 +47,48 @@
...
+
+// SKYRAT EDIT BEGIN -- ONE CLICK ANTAG
+	else if(href_list["makeAntag"])
+
+		message_admins("[key_name_admin(usr)] is attempting to make [href_list["makeAntag"]]")
+
+		if(!check_rights(R_ADMIN))
+			return
+
+		if (!SSticker.mode)
```
Comment (JohnFulpWillard) [inline code/modules/admin/topic.dm:59]: Isn't gamemode being deleted soon?

**Comment by hugmeorelse** [comment]
@Gandalf2k15 @GoldenKeyboard I notice this is not TMed or merged yet. Any reason?

**Comment by ZephyrTFA** [comment]
> 
> 
> @Gandalf2k15 @GoldenKeyboard I notice this is not TMed or merged yet. Any reason?

It was testmerged; but I accidentally deleted my branch so TGS went "haha, fuk u"

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (ZephyrTFA) [reply code/modules/admin/topic.dm:59]: SSticker.mode is used to store if a round has started, and was also used to store the mode itself; all that does is check if the game has started

**Comment by ZephyrTFA** [comment]
Alright; I some issues I need to fix; but it shouldn't take long.

**Comment by Gandalf2k15** [comment]
LGTM

**Pull request merged**